web/templates/shared/flashes.html.eex

```diff
@@ -0,0 +1,9 @@
+<%= for flash_key <- [:info, :error] do %>
+  <%= if flash = get_flash(@conn, flash_key) do %>
+    <div class="flash flash-<%= flash_key %>">
+      <div class="container">
+        <p data-role="flash"><%= raw flash %></p>
```

Comment:
Sounds good :)
> On Apr 22, 2016, at 3:39 PM, Jason Draper <notifications@github.com> wrote:
> 
> In web/templates/shared/flashes.html.eex <https://github.com/drapergeek/community/pull/24#discussion_r60792528>:
> 
> > @@ -0,0 +1,9 @@
> > +<%= for flash_key <- [:info, :error] do %>
> > +  <%= if flash = get_flash(@conn, flash_key) do %>
> > +    <div class="flash flash-<%= flash_key %>">
> > +      <div class="container">
> > +        <p data-role="flash"><%= raw flash %></p>
> I think for now I'll omit the raw until we decide we want it again.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/drapergeek/community/pull/24/files/5238dc5f2e286ee8cbd51e367bed6ef36dd7e645#r60792528>